SCons/Tool/VCPkgTests.py

```diff
@@ -0,0 +1,528 @@
+#
+# __COPYRIGHT__
```

Comment:
For new files, please follow the templates in `templates/file.py` and `templates/test.py` - some files have not yet been converted, in particular Tool modules, so realize you probably got this form from an existing file...